img/b/berkeley.edu.svg

```diff
@@ -0,0 +1 @@
1.46-40.09 6.56 19.06 99.1 51.97 130.62 64.99 6.67-13.63 28.13-38.61 28.13-44.36 12.12 4.84 15.04 18.92 26.56 24.68-23.03 16.68-54.75 72.04-56.88 97.5M2344.32 1701.23c10.3-17.58 23.14-16.1 35.87-28.83-29.1-11.52-51.03-27.55-70.43-46.95 10.91-10.91 14.66-24.16 30.43-39.92 22.12 10.91 60.3 50.13 60 54.37 12.12-18.19 6.57-27.17 19.3-39.3-15.76 1.81-139.43-104.88-139.43-118.82-10.92 14.55-25.5 35.62-25.5 35.62 23.04 12.73 38.9 23.32 63.75 50-7.27 9.7-17.65 22.29-32.19 42.89-13.34-4.84-63.05-34.24-63.05-52.12-14.55 11.51-17.88 17.98-36.63 27.98 43.65 21.83 153.33 93.86 157.88 115.08M2445.5 1576.78c.61-5.46-88.59-124.53-113.75-149.69 25-11.25 97.44-46.66 115.62-69.69 12.13 10.91 26.83 32.23 26.83 32.23-5.16-1.51-71.54 30.31-92.15 48.5 12.12 16.98 16.75 21.67 32.51 38.65 18.19-14.55 31.59-13.73 42.51-33.13 2.42 3.64 14.52 14.75 19.37 26.88-21.83 12.12-31.88 17.5-45.62 31.25 3.63 8.49 13.97 23.64 25.19 33.35 33.35-22.13 46.06-25.23 67.93-50.23 6.06 14.54 3.75 15 12.5 34.38-41.25 17.5-48.75 23.12-90.94 57.5M2701.75 1463.03l118.44 9.37 9.37-33.43c-8.12 5.62-54.37 3.12-90.62-1.57l1.87-40c20.2-2.56 43.13 2.19 51.25 5l3.75-29.37c-14.37 4.06-29.06-3.13-54.37-3.13l-.63-51.25c23.33 0 79.38-1.25 92.5 17.5l-5.62-40s-57.5-17.5-123.13-2.81c9.96 5.97 9.56 72.96 8.43 87.19-2.56 32.7 3.13 55.31-11.24 82.5M3075.19 1648.65c13.75-3.75 19.73-7.47 29.73-8.09-11.25-28.75-10.94-35-28.44-52.5 10.86-7.6 20.63-20.31 26.88-26.56 14.37 13.75 18.44 26.25 20.94 35 8.75-8.13 13.43-15 22.18-16.88-5.62-20-9.37-26.25-23.75-38.75 11.25-15 12.82-15.62 23.13-26.25 26.25 11.88 37.19 54.06 37.19 54.06s14.64-24.4 18.39-28.78c-10.63-35-38.75-50-62.5-66.25-27.5 40-89.49 111.36-108.75 110 32.5 27.5 36.87 30.63 45 65M3498.43 2101.4c-36.15 7.48-88.64-14.87-97.09-80.7-6.37-49.7 26.63-97.46 61.49-104.03 58.44-11.01 69.95 42.38 73.34 77.96.6 6.18 7.06 8.04 7.1 13.63-6.75.09-47.67 12.86-61.65 14.74-1.33-11.95-.68-12.49-6.6-26.03 17.87-1.39 23.71-4.46 33.38-9.51.16-5.13-1.73-52.08-36.93-42.14-29.78 8.41-44.77 29.08-40.45 74.43 3.6 37.85 31.16 63.09 61.25 57.44 20.88-3.93 32.34-12.74 32.34-25.86 14.76 7.21 25.37 15.52 25.37 15.52-14.98 22.36-18.31 27.66-51.55 34.55M3614.25 2093.65c-2.19-23.76.73-52.38 3.75-68.12 24.29 8.56 41.12 12.05 66.83 6.34 0 0-5.93 69.32-12.14 71.47 7.27 4.24 36.87-5 36.87-5-2.81-4.69-4.37-10.94-4.1-14.68 3.14-44.34 13.52-153.32 23.48-155.33-3.64-1.81-18.04 2.6-33.75-3.11 3.03 4.23-7.24 61.76-8.75 78.75-22.87 12.51-51.61-.68-67.06-3.97 0-10.01.18-71.67 9.87-72.91-19.09-7.58-21.1-3.17-41.41-4.69 0 3.64 5.53 9.6 6.07 15.77 1.28 14.54-8.98 142.69-17.16 146.72 0 0 29.69 3.14 37.5 8.76M3746.62 2090.26c39.32.93 59.03-1.55 90 18.12 10.94-15.31 12.5-18.44 22.19-28.44-13.64.91-37.19-7.5-49.38-11.87 15.74-67.09 33.13-125.31 38.44-127.81-15.16-4.55-21.08-5.82-35.62-11.88-1.57 35.94-24.38 106.88-30.63 135.63-22.5-3.13-30.31-4.38-46.56-12.19-.92 2.72 7.23 32.2 11.56 38.44" style="fill:#c57f0a;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 709.893)"/></svg>
```

Comment:
omg look at the file size 😓

In this case a PNG would be preferred :wink:

Reply:
I'll trim it down